Sources/Turf/Geometries/Point.swift

```diff
@@ -5,9 +5,9 @@ import CoreLocation
 
 
 public struct Point: Equatable {
-    public var coordinates: CLLocationCoordinate2D
+    public var coordinate: CLLocationCoordinate2D
```

Comment:
The reasoning behind this name in the first place was that according to the [GeoJSON RFC](https://tools.ietf.org/html/rfc7946#section-1.5), a Point’s coordinate is serialized as `coordinates` though everything but the first coordinate is being ignored.

Reply:
Oh, okay, that makes sense. I've reset this PR and added that information to the incline documentation so others might not try to do it again in the future. How does that sound?